files/ru/learn/css/building_blocks/the_box_model/index.html

```diff
@@ -263,13 +265,13 @@ <h3 id="Границы">Границы</h3>
  <li>{{cssxref("border-left-color")}}</li>
 </ul>
 
-<p><strong>В примере ниже мы использовали различные сокращённые и длинные способы создания границ. Поиграйте с различными свойствами чтобы проверить как вы поняли принципы их работы. MDN страницы свойств границ дадут вам информацию о различных стилях, которые вы можете использовать.</strong></p>
+<p><strong>В примере ниже мы использовали различные сокращённые и длинные способы создания рамок. Поиграйте с различными свойствами чтобы проверить как вы поняли принципы их работы. MDN страницы свойств рамок дадут вам информацию о различных стилях, которые вы можете использовать.</strong></p>
```

Comment:
```suggestion
<p><strong>В примере ниже мы использовали различные сокращённые и полные способы создания рамок. Поиграйте с различными свойствами, чтобы проверить, как вы поняли принципы их работы. Информацию о различных стилях, которые вы можете использовать, можно найти на страницах MDN о свойствах рамок.</strong></p>
```
Переписала последнее чуть более по-русски. В оригинале там:
`The MDN pages for the border properties give you information about the different styles of border you can choose from.`